src/index.html

```diff
@@ -11,7 +11,152 @@
+        <label class="form-label">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-label">
+          Name:
+          <input
+          type="text"
+          name="name"
+          id="name"
+          autocomplete="off"
+          minlength="3"
+          maxlength="10"
+          >
+        </label>
+        <label class="form-label">
+          How old are You?
+          <input
+          type="number"
+          name="years-old"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+        <label class="form-label">
+          Full date of birth:
+          <input
+          type="date"
+          name="dateOfBirth"
+          min="1923-01-01"
+          max="2022-01-01"
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="registration">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label class="registration1">
+          Password:
+          <input
+          type="password"
+          name="password"
+          id="password"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="facts">
+          Do you love cats?
+          <label class="radio-label">
+            <input
+            type="radio"
+            name="cat"
+            value="Yes"
+            required
+            >
+            Yes
+          </label>
+          <label class="radio-label">
+            <input
+            type="radio"
+            name="cat"
+            value="No"
+            required
+            >
+            No
+          </label>
+        </label>
+        <label class="facts">
+         What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          required
+          >
+        </label>
+        <label class="facts">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          required
+          >
+        </label>
+        <label class="facts">
+          What are your favorite brand of cars?
+          <select multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="facts1">
+          How do you rate our work?
+          <input type="range" value="0">
```

Comment:
Add the 'name' attribute to the 'input' element with type='range' to include it in the form data 📝